samples/bluetooth/peripheral_uart_nrf53/application_core/src/main.c

```diff
@@ -0,0 +1,323 @@
+/*
+ * Copyright (c) 2020 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+
+/** @file
+ *  @brief Nordic UART Bridge Service (NUS) sample
+ */
+
+#include <zephyr/types.h>
+#include <zephyr.h>
+#include <uart.h>
+
+#include <device.h>
+#include <soc.h>
+
+#include <dk_buttons_and_leds.h>
+
+#include <stdio.h>
+#include <string.h>
+#include <init.h>
+
+#include "bt_ser.h"
+#include "rpmsg.h"
+
+#define STACKSIZE CONFIG_BT_GATT_NUS_THREAD_STACK_SIZE
+#define PRIORITY 7
+
+#define RUN_STATUS_LED DK_LED1
+#define RUN_LED_BLINK_INTERVAL 1000
+
+#define CON_STATUS_LED DK_LED2
+
+#define UART_BUF_SIZE CONFIG_BT_GATT_NUS_UART_BUFFER_SIZE
+
+#define BT_ADDR_LE_STR_LEN 30
+
+static K_SEM_DEFINE(ble_init_ok, 0, 2);
+
+static struct device *uart;
+
+struct uart_data_t {
+	void *fifo_reserved;
+	u8_t data[UART_BUF_SIZE];
+	u16_t len;
+};
+
+static K_FIFO_DEFINE(fifo_uart_tx_data);
+static K_FIFO_DEFINE(fifo_uart_rx_data);
+
+static int bt_addr_le_to_str(const bt_addr_le_t *addr, char *str,
```

Comment:
For me serialization for this function does not have any sense. It extract struct with address to string. So can be call on default place. I write it here because I am not able to include it from Bluetooth because there is to many dependecies to other subsystems